<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>oafrontend</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>481</width>
     <height>584</height>
    </rect>
   </property>
   <widget class="QListView" name="lstGameView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbViewVcs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Game Info</string>
    </property>
    <widget class="QLabel" name="lblName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblController">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Controller:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCompany">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Company:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblScreen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Screen:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Rom File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNameValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>261</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblControllerValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>251</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCompanyValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>271</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblScreenValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>271</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRomValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leRomFile">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbScreen">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbCompany">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbController">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblMapper">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Mapper:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMapperValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>251</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbMapper">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSuperChip">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SC</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSuperChip">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>SC</string>
     </property>
    </widget>
    <zorder>lblName</zorder>
    <zorder>lblController</zorder>
    <zorder>lblCompany</zorder>
    <zorder>lblScreen</zorder>
    <zorder>lblRom</zorder>
    <zorder>lblControllerValue</zorder>
    <zorder>lblCompanyValue</zorder>
    <zorder>lblScreenValue</zorder>
    <zorder>lblRomValue</zorder>
    <zorder>leName</zorder>
    <zorder>leRomFile</zorder>
    <zorder>lblNameValue</zorder>
    <zorder>cmbScreen</zorder>
    <zorder>cmbCompany</zorder>
    <zorder>cmbController</zorder>
    <zorder>lblMapper</zorder>
    <zorder>lblMapperValue</zorder>
    <zorder>cmbMapper</zorder>
    <zorder>cbSuperChip</zorder>
    <zorder>lblSuperChip</zorder>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>530</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
